WORKSPACE

```diff
@@ -93,6 +93,18 @@ http_archive(
     url = "https://github.com/google/tcmalloc/tarball/cf3dc2d98bd64cb43f4f98db0acaf5028a7b81eb",
 )
 
+git_repository(
+    name = "dragonbox",
+    build_file_content = """cc_library(
+            name = "dragonbox",
+            hdrs = glob(["include/dragonbox/*.h"]),
+            visibility = ["//visibility:public"],
+            include_prefix = "third_party/dragonbox/include/dragonbox",
```

Comment:
@anonrig I think this should just be `third_party/dragonbox ` since the `include/dragonbox` path already exists in the dragonbox repo.

Reply:
Thanks for the review. I've updated this.

Reply:
This doesn't seem to work.

```
Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
external/v8/src/numbers/conversions.cc:26:10: fatal error: 'dragonbox.h' file not found
   26 | #include "dragonbox.h"
      |          ^~~~~~~~~~~~~
1 error generated.
```

What's the correct path for the include statement here?

Reply:
The version you just pushed should work – if you want to go without patching conversions.cc it should be `include_prefix = third_party/dragonbox/src` but either works.